tests/v2/validation/provisioning/rke2/README.md

```diff
@@ -55,7 +55,7 @@ provisioningInput:
   rke2KubernetesVersion: ["v1.27.10+rke2r1"]
   cni: ["calico"]
   providers: ["linode", "aws", "do", "harvester", "vsphere", "azure"]
-  cloudProvider: "aws"
+  cloudProvider: "aws" # rancher-vsphere
```

Comment:
nit - can you update the comment to say something like `For vSphere, enter rancher-vsphere`